### ISSUE TITLE
in_node_exporter_metrics: broken timestamp for node_systemd_system_running

### ISSUE THREAD

**Issue by positron96** [fluent/fluent-bit#7621]
I am using fluent-bit 2.1.6 on Linux (Raspberry Pi, Raspbian 11) and have configured node-explorer input with mostly default settings:
```
pipeline:  
  inputs: 
    - name: node_exporter_metrics 
      tag:  metrics_node  
      scrape_interval: 60 
  outputs:
    - name: stdout  
      match: "metrics*"   
      format: json_lines  
 ```

The metrics seem good, except the node_systemd_system_running is showing zero timestamp. Here is an output from stdout (note the last line):
```
2023-06-29T11:57:12.242099089Z node_boot_time_seconds = 1688023927
2023-06-29T11:57:12.242099089Z node_procs_running = 3
2023-06-29T11:57:12.242099089Z node_procs_blocked = 0
2023-06-29T11:57:12.242278326Z node_time_seconds = 1688039832.2422783
2023-06-29T11:57:12.242409602Z node_load1 = 0.17000000000000001
2023-06-29T11:57:12.242409602Z node_load5 = 0.17000000000000001
2023-06-29T11:57:12.242409602Z node_load15 = 0.16
2023-06-29T11:57:12.244408198Z node_filefd_allocated = 3968
2023-06-29T11:57:12.244408198Z node_filefd_maximum = 2147483647
1970-01-01T00:00:00.000000000Z node_systemd_system_running = 0
```

As a consequence, the prometheus remote_write server (not shown in config) complains about timestamp being too old:
```
 [output:prometheus_remote_write:prometheus_remote_write.3] prometheus-prod-22-prod-eu-west-3.grafana.net:443, HTTP status=400
failed pushing to ingester: user=XXX: the sample has been rejected because its timestamp is too old (err-mimir-sample-timestamp-too-old). The affected sample has timestamp 1970-01-01T00:00:00Z and is from series {__name__="node_systemd_system_running"}
```

**Comment by positron96** [comment]
@helmut72 to me this seems like a different issue, but idk.

**Comment by helmut72** [comment]
Maybe you are right, but I get the message even if I delete my metrics data in Mimir and start from scratch. They can't be out-of-order if no data exists. But I think I should open an own issue.

**Comment by helmut72** [comment]
You were right:
https://github.com/fluent/fluent-bit/issues/7763#issuecomment-1654580965

**Comment by cianbe** [comment]
We're experiencing a similar issue. We included `systemd` in `node_exporter_metrics` [INPUT] in fluent-bit.conf to capture system unit states in Grafana. When we observed stdout and logs, we also saw zero timestamp for node_systemd_system_running:
_1970-01-01T00:00:00.000000000Z node_systemd_system_running = 0_
In addition to this, metrics stopped flowing into Grafana.
We upgraded to fluent-bit 2.2.2 thinking it would be fixed as this issue looked similar to https://github.com/fluent/fluent-bit/issues/8368. However, issue still persists.
